internal/pkg/util/fs/squashfs/squashfs.go

```diff
@@ -53,23 +55,35 @@ func GetPath() (string, error) {
 }
 
 func GetProcs() (uint, error) {
+	// prioritize the environment variable over config
+	procstr, exists := os.LookupEnv("SINGULARITY_MKSQUASHFS_PROCS")
+	if exists {
+		proc, err := strconv.ParseUint(procstr, 10, 32)
+		return uint(proc), err
```

Comment:
Please return an error here that makes it clear the problem is coming from the env var, so it can't be confused with a `singularity.conf` or other issue.